Content.Shared/Humanoid/Markings/MarkingsSet.cs

```diff
@@ -163,6 +181,25 @@ public void FilterSpecies(string species, MarkingManager? markingManager = null,
         {
             Remove(remove.category, remove.id);
         }
+
+        // Re-color left markings them into skin color if needed (i.e. for slimes)
+        if (skinColor != null)
+        {
+            foreach (var (category, list) in Markings)
+            {
+                foreach (var marking in list)
+                {
+                    if
+                    (
```

Comment:
code style (please put the first parenthesis, and the first item in that if condition on the same line as the if keyword)